contracts/modules/dispute/policies/ArbitrationPolicySP.sol

```diff
@@ -74,8 +85,12 @@ contract ArbitrationPolicySP is IArbitrationPolicy, Governable {
     /// @dev Enforced to be only callable by the governance protocol admin.
     function governanceWithdraw() external onlyProtocolAdmin {
         uint256 balance = IERC20(PAYMENT_TOKEN).balanceOf(address(this));
-        IERC20(PAYMENT_TOKEN).safeTransfer(governance, balance);
+        IERC20(PAYMENT_TOKEN).safeTransfer(msg.sender, balance);
```

Comment:
Should we add a protocol-wide variable with the treasury address, where all present and future flows go to?

Reply:
Seems like a good idea. Noted!